app/routes/admin/reports/system-logs/mail-logs.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+
+const { Route, String } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    return this.i18n.t('Mail Logs');
+  },
+
+  model() {
+    return [
+      {
+        recipient : 'sample@gmail.com',
+        time      : new Date(),
+        action    : 'Session Accept or Reject',
+        subject   : 'Session test proposal has been accepted',
+        message   : String.htmlSafe('Hi y@z.com,<br/>The session <strong>test proposal </strong> has been <strong>accepted</strong> by the organizer. <br/> Visit this link to view the session: http://localhost:5000/events/6/sessions/14/')
```

Comment:
@CosmicCoder96 @geekyd Emails will remain as it is. As for the notifications, that will change. But since that is out of the scope of this ticket, let it be like this for now. I'll open a new issue for it once the response format is finalised on the BE.

Tentative response format for notifications would be something like:
```js

{
    "title": "title",
    "description": "The description/sub-title",
    "positiveAction": "url to that action",
    "negativeAction": "url to that action",
    "secondaryAction": "url to that action"
}
```